<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>
 <HEAD>
   <TITLE> [mod_python] Sorry, Install error.
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20Sorry%2C%20Install%20error.&In-Reply-To=">
   <META NAME="robots" CONTENT="index,nofollow">
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="011744.html">
   <LINK REL="Next"  HREF="011749.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[mod_python] Sorry, Install error.</H1>
<table border=0 width="100%"><tr><td valign="top">
    <B>nineclue at mrham.hamslab.com</B> 
    <A HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20Sorry%2C%20Install%20error.&In-Reply-To="
       TITLE="[mod_python] Sorry, Install error.">nineclue at mrham.hamslab.com
       </A><BR>
    <I>Tue Apr  3 21:32:45 EST 2001</I>
    <P><UL>
        <LI>Previous message: <A HREF="011744.html">[mod_python] Install Error
</A></li>
        <LI>Next message: <A HREF="011749.html">[mod_python] Apache configuration
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#11748">[ date ]</a>
              <a href="thread.html#11748">[ thread ]</a>
              <a href="subject.html#11748">[ subject ]</a>
              <a href="author.html#11748">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Greetings....

Sorry for the contamination.
Now I'm using PINE thru telnet.

I've compiled Apache with mod_python using ApacheToolbox. When I tried
the Test (mptest.py) script, Server Error is raised. 
Followings are part of error_log.

[Tue Apr  3 21:35:23 2001] [notice] Apache/1.3.19 (Unix)
mod_gzip/1.3.19.1a mod_python/2.7.2 Python/2.0 mod_ssl/2.8.1 OpenSSL/0.9.5a configured --
resuming normal operations
[Tue Apr  3 21:35:23 2001] [notice] suEXEC mechanism enabled (wrapper: 
/usr/local/apache/bin/suexec)
'import site' failed; use -v for traceback
make_obcallback(): could not import mod_python.apache.
make_obcallback(): could not call init.
[Tue Apr  3 21:35:44 2001] [error] [client 127.0.0.1] python_handler:
make_oballback returned no obCallBack!

Any comment or answer would be appreciated.

Suhku Huh



</PRE>
<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="011744.html">[mod_python] Install Error
</A></li>
	<LI>Next message: <A HREF="011749.html">[mod_python] Apache configuration
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#11748">[ date ]</a>
              <a href="thread.html#11748">[ thread ]</a>
              <a href="subject.html#11748">[ subject ]</a>
              <a href="author.html#11748">[ author ]</a>
         </LI>
       </UL>

</td>
<td align="right" valign="top">
<script type="text/javascript"><!--
google_ad_client = "pub-9718360309690383";
google_ad_width = 120;
google_ad_height = 600;
google_ad_format = "120x600_as";
google_color_border = "CCCCCC";
google_color_bg = "FFFFFF";
google_color_link = "000000";
google_color_url = "666666";
google_color_text = "333333";
//--></script>
<script type="text/javascript"
  src="DISABLEhttp://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</td>
</table>

<hr>
<a href="http://mailman.modpython.org/mailman/listinfo/mod_python">More information about the Mod_python
mailing list</a><br>
</body></html>
